lib/crypto/src/crypto.erl

```diff
@@ -426,6 +426,7 @@
 -type sha3() :: sha3_224 | sha3_256 | sha3_384 | sha3_512 .
 -type sha3_xof() :: shake128 | shake256 .
 -type blake2() :: blake2b | blake2s .
+-type sm3() :: sm3 .
```

Comment:
Don't see the point with a type containing one atom. Unless we anticipate more. I saw that sm2 and sm4 exist as well, but they are not hashes.

Reply:
already remove sm3 type in the new commit